cmd/transcriber/call/tracks.go

```diff
@@ -181,7 +189,7 @@ func (t *Transcriber) processLiveTrack(track trackRemote, sessionID string) {
 		}
 
 		var gap uint64
-		if ctx.startTS == 0 {
```

Comment:
Changing this since, in theory, `startTS` could be still 0 even after setting it the first time if we are very unlucky and the start time was set just now. Very unlikely but you never know :)